### PR TITLE
Delete features moved to the SonarQube extension

### PR DESCRIPTION
**Pull request by henryju**
This pull request is the consequence of the last release of the new SonarQube and SonarCloud VSTS extensions. Configuration of the SonarQube analysis for Maven/Gradle as well as the analysis report publishing in VSTS dashboard are now handled by those extensions.

Support of pull request decoration will come later (probably as a SonarQube server side feature).

### CODE REVIEW

File: Tasks/Gradle/task.json [hunk 1]
```diff
@@ -227,126 +228,30 @@
...
-            "groupName": "CodeAnalysis",
-            "visibleRule": "sqAnalysisEnabled = true"
+            "helpMarkDown": "Run a [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing the current goals. 'install' or 'package' goals should be executed first."
         },
         {
             "name": "sqGradlePluginVersion",
             "aliases": [ "sonarQubeGradlePluginVersion" ],
-            "type": "string",
-            "label": "SonarQube Gradle Plugin Version",
+            "type": "pickList",
```
Comment (madhurig) [inline Tasks/Gradle/task.json]: PickList seems like a bad idea for version which might change going forward. You can have a radio button for "Use plugin applied in your build.gradle" or "Specify version number" and default the the specific version number to 2.6.1

Comment (henryju) [reply Tasks/Gradle/task.json]: Good idea!

File: Tasks/Gradle/task.json [hunk 2]
```diff
@@ -14,10 +14,11 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 1,
-        "Minor": 128,
+        "Major": 2,
+        "Minor": 0,
```
Comment (madhurig) [inline Tasks/Gradle/task.json]: Please update minor version to "130"

File: Tasks/Maven/task.json [hunk 3]
```diff
@@ -15,10 +15,11 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 1,
-        "Minor": 128,
+        "Major": 2,
+        "Minor": 0,
```
Comment (madhurig) [inline Tasks/Maven/task.json]: Please update minor version to 130

File: Tasks/Gradle/task.json [hunk 4]
```diff
@@ -227,125 +228,36 @@
             "name": "sqAnalysisEnabled",
             "aliases": [ "sonarQubeRunAnalysis" ],
             "type": "boolean",
-            "label": "Run SonarQube Analysis",
+            "label": "Run SonarQube or SonarCloud Analysis",
             "required": true,
             "defaultValue": "false",
             "groupName": "CodeAnalysis",
-            "helpMarkDown": "Run a [SonarQube analysis](https://go.microsoft.com/fwlink/?LinkID=708598) after executing the current goals. 'install' or 'package' goals should be executed first."
+            "helpMarkDown": "Run a [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing the current tasks. You have to add the `Prepare Analysis Configuration` task of the [SonarQube](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube) or [SonarCloud](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) extensions in your build definition before this Gradle task."
```
Comment (madhurig) [inline Tasks/Gradle/task.json]: @DavidStaheli : Any feedback/suggestions on this help text?

File: Tasks/Maven/task.json [hunk 5]
```diff
@@ -274,112 +275,23 @@
             "name": "sqAnalysisEnabled",
             "aliases": [ "sonarQubeRunAnalysis" ],
             "type": "boolean",
-            "label": "Run SonarQube Analysis",
+            "label": "Run SonarQube or SonarCloud Analysis",
             "required": true,
             "defaultValue": "false",
             "groupName": "CodeAnalysis",
-            "helpMarkDown": "Run a [SonarQube analysis](https://go.microsoft.com/fwlink/?LinkID=708598) after executing the current goals. 'install' or 'package' goals should be executed first."
+            "helpMarkDown": "Run a [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing the current goals. `install` or `package` goals should be executed first. You have to add the `Prepare Analysis Configuration` task of the [SonarQube](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube) or [SonarCloud](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) extensions in your build definition before this Maven task."
```
Comment (DavidStaheli) [inline Tasks/Maven/task.json]: Can we please change this as follows to explain how the behavior changed from version 1 of the Maven task?

```markdown
This option has changed from version 1 of the **Maven** task to use the [SonarQube](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube) and [SonarCloud](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) marketplace extensions.  Enable this option to run [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing goals in the **Goals** field. The **install** or **package** goal should run first. You must also add a **Prepare Analysis Configuration** task from one of the extensions to the build definition before this Maven task.
```

File: Tasks/Gradle/task.json (hunk 4 above)
Comment (DavidStaheli) [reply Tasks/Gradle/task.json]: Can we please change this as follows to explain how the behavior changed from version 1 of the Gradle task?

```markdown
This option has changed from version 1 of the **Gradle** task to use the [SonarQube](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube) and [SonarCloud](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) marketplace extensions.  Enable this option to run [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing tasks in the **Tasks** field. You must also add a **Prepare Analysis Configuration** task from one of the extensions to the build definition before this Gradle task.
```

**Pull request merged**